File: 1. Conditional Problems/Conditional Problems in Python/p10.py
#  write a python program to input month number and print number of days in that month

m = int(input("Enter month number: "))

if m == 1 or m == 3 or m == 5 or m == 7 or m == 8 or m == 10 or m == 12:
    print("31 Days")
elif m == 4 or m == 6 or m== 9 or m == 11:
    print("30 days")
elif m == 2:
    print("28 or 29 days")

File: 1. Conditional Problems/Condional Problems in C/p10.c
// write a C program to input month number and print number of days in that month

#include <stdio.h>

int main() {
    int m;

    
    printf("Enter Month number: ");
    scanf("%d", &m);

    
  
    if (m == 1 || m == 3 || m == 5 || m == 7 || m == 8 || m == 10 || m == 12) {
        printf("31 days");
    }
    else if (m == 4 || m == 6 || m== 9 || m == 11) {
        printf("30 days");
    }
    else if (m == 2) {
        printf("28 or 29 days"); }
    else {
        printf("Invalid month number");
    }

    
}
